ext/standard/config.w32

```diff
@@ -22,13 +22,15 @@ AC_DEFINE("PHP_USE_PHP_CRYPT_R", 1);
 
 CHECK_HEADER_ADD_INCLUDE("timelib_config.h", "CFLAGS_STANDARD", "ext/date/lib");
 
+ADD_FLAG("LIBS_STANDARD", "iphlpapi.lib");
```

Comment:
@weltling tho we only use iphlpapi here, wouldn't you think its better put in toolset_setup_common_libs() in win32/confutils.js, ext/standard can't be disabled anyway

Reply:
@KalleZ would work as well, but shouldn't be necessary. `$(LIBS)` is put for every binary unit, whereby `iphlpapi.lib` is only required for standard. The linker will ignore it anyway, if there are no symbols used.

Reply:
It sounds like six of  one, half a dozen of the other.  I honestly don't mind either way, but I read @weltling's comment (and @KalleZ's 👍) as "don't bother moving it.  Correct me if that's wrong.